src/directives/Tooltip/index.js

```diff
@@ -3,13 +3,18 @@
  * SPDX-License-Identifier: AGPL-3.0-or-later
  */
 
-import { VTooltip, options } from 'floating-vue'
+import { vTooltip, options } from 'floating-vue'
 import './index.scss'
 
 options.themes.tooltip.html = false
 options.themes.tooltip.delay = { show: 500, hide: 200 }
 options.themes.tooltip.distance = 10
 options.themes.tooltip['arrow-padding'] = 3
 
-export default VTooltip
-export { options }
+export {
+	/**
+	 * @deprecated Use the native `title` attribute instead.
```

Comment:
In some cases NcPopover would be a good alternative, in case tooltip has some complex logic or styling inside ( so I'd mention it here)

Reply:
Probably because it was only us until 29 releases, maybe that's why =D

directive basically [allows same props](https://floating-vue.starpad.dev/api/#directive-options) as VDropdown component, so for heavy customization someone might consider it (we used `delay` back then, for example)

Reply:
added documentation

Reply:
Moving here from the follow-up PR: `NcRichContenteditable` is exactly the case where another solution should be used, and not the title visible on hover.

If maxlength exceeded, it's not shown anymore to the end-user, thus hiding error visibility

B | A
-- | --
<img width="754" alt="image" src="https://github.com/user-attachments/assets/fda3de6a-1e51-448c-9294-0c63a4839b7c" /> | <img width="810" alt="image" src="https://github.com/user-attachments/assets/a4fdaf2f-1113-47e4-ae0f-2507cfa0a3d4" />